File: Data Structures & Algorithms/is-anagram/submission-1.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.size() != t.size()){
            return false; 
        }
        vector<int>m(26, 0); 
        vector<int>n(26, 0);
        for(int i= 0 ; i < s.size() ; i++){
        m[s[i]-'a']++;
        n[t[i]-'a']++;

    }
     if(m!= n ){
            return false; 
        }
        
    return true ;
    }
};

File: Data Structures & Algorithms/longest-common-prefix/submission-1.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        strs.sort()
        n = len(strs)
        prefix_string = strs[0]
        last_string = strs[n - 1]
        j = 0
        while j < min(len(prefix_string), len(last_string)) and prefix_string[j] == last_string[j]:
            j += 1
        return prefix_string[0:j]